### DISCUSSION TITLE
Profile Social Links?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#37683] [Other Feature Feedback, Questions, & Ideas] [answered]
At the moment, GitHub profiles only offer the option to link to a Twitter account.

![Screenshot_20221028_210251](https://user-images.githubusercontent.com/62166915/198752188-8bcbe6d0-5e78-4911-b232-12efcba2ce14.png)

Can more be added? Or change it to a user-configurable table like this:

|||
|-|-|
| Mastodon | https://mastodon.social/username |
| Pixelfed | https://pixelfed.social/username |
| Reddit | https://reddit.com/u/username |

For example, it shows up like this on Mastodon (last two items):

![Screenshot_20221028_210813](https://user-images.githubusercontent.com/62166915/198752500-7965e126-56a3-4cda-ac3f-fabc0fd70374.png)

**Comment by rpriyanshu9** [comment]
No, not at the moment. Maybe we can raise an a feature request for the same. But as of now, I don't think so any such feature is in place.

**Comment by jfullstackdev** [comment]
use the default README on your landing page, you can even use external services for extra pins,

this is my landing page using an external GitHub Stats service

![image](https://user-images.githubusercontent.com/47092464/198882900-de38e1d9-c6f6-4216-a67e-0623ebd07eb1.png)

the external service is this 

https://github.com/anuraghazra/github-readme-stats

**Reply by corneliusroemer** [reply]
This is not the same. In particular, GitHub does not add that rel=me tag that would allow mastodon to show a green tick.

**Reply by babbage** [reply]
If you put a link to your Mastodon user profile page into the website field of your GitHub profile, and on Mastodon edit and save your Mastodon profile page (which prompts checking link verification), your GitHub account will receive the Mastodon green check verification. After this, you can change the website link back on your GitHub profile and in my testing you will continue to have the green check verification on Mastodon, even after subsequent profile edits.

It would still be great to have the option to list a Mastodon social link on a GitHub profile page instead of Twitter, but fortunately it is already possible to achieve the green check verification anyway.

**Reply by sunarch** [reply]
I have read in an article about this topic that verification is triggered separately for other instances, so that if you change the link immediately only your own instance will show the verification icon. I don't know if this is still the case, but if your instance ever re-validates the cache, it would also disappear.

**Reply by babbage** [reply]
Interesting @sunarch. That makes perfect sense.

**Comment by corneliusroemer** [comment]
I wanted to put in this request and found yours, thanks!

Can you add to your request that the link should have the correct rel=me tag so that mastodon can add that green tick?

See link verification here https://docs.joinmastodon.org/user/profile/
![Screenshot_2022-11-09-13-34-01-14_e4424258c8b8649f6e67d283a50a2cbc.jpg](https://user-images.githubusercontent.com/25161793/200831627-ad9fb974-d89c-410b-86dd-5a3eb1742930.jpg)

**Comment by david-engelmann** [comment]
[Additional Discussion #1](https://github.com/community/community/discussions/5720)

[Additional Discussion #2](https://github.com/community/community/discussions/38488)

[Additional Discussion #3](https://github.com/community/community/discussions/38560)

**Comment by ghost** [comment]
You can't link, but you can always leave your LinkedIn profile URL in your profile, so somebody interested can find you there, and interchangeable, leave a GitHub link on LinkedIn and so forth

**Reply by Lupe520** [reply]
Question..how would I leave my GitHub link on LinkedIn or so I can put on my resume??

**Comment by Alkebsi** [comment]
Check https://shields.io/ 

I have added some links to my default README file, you can see how it works. Also, this repo have helped me along the way - https://github.com/alexandresanlim/Badges4-README.md-Profile

**Reply by Alkebsi** [reply]
Wish that helps you!

**Comment by babbage** [comment]
Getting a green check on Mastodon to verify your GitHub identity is already possible, using the website link section in your GitHub profile. See [this simple guide](https://jan.wildeboer.net/2022/06/Quick-Fix-Verified-Mastodon-Github/).

It would still be great to have an option to list a Mastodon profile in the main GitHub profile links, instead of (or as well as) a Twitter profile. 👍 

<img width="573" alt="Mastodon profile page of @babbage @iosdev.space showing verified links for intro.app and his GitHub profile" src="https://user-images.githubusercontent.com/376937/202020353-a12b7f0a-ff81-47f7-b013-7edb76313d88.png">

**Reply by sunarch** [reply]
But if you want to link to an actual website there -- as many people do -- and don't want to sacrifice that slot, there are currently no other options. (Except using the GitHub Pages domain, but that's an alternative, not a solution.)

**Comment by david-engelmann** [comment]
Mastodon should update their verification process to reflect it's no longer 2018 before Github should start hard coding social networks to the profile section

**Reply by babbage** [reply]
🔥😝 though to be fair, as indicated above, Mastodon already provides a way for GitHub accounts to be verified. It's GitHub that has hardcoded Twitter as a profile field and doesn't provide for the alternative.

**Reply by david-engelmann** [reply]
Hijacking the website section of your Github profile isn't that great of a workaround. These complaints would be better served by Mastodon. Github can't change for ever social media platforms blue check system. rel="me" isn't supported in the Readme for a reason

**Reply by sunarch** [reply]
@david-engelmann What is the reason to disallow rel="me" in the profile ReadMe? I'm guessing it's preventing some kind of problem, but I can't currently think of what that could be.

**Comment by derekprior** [comment]
Hey folks, I just wanted to give you a heads up that we are working on this. It'll ship when it's ready, but with holiday season vacations coming up that won't be for a bit yet. Thanks for your patience.

**Reply by sunarch** [reply]
Thank you for the update and the work!

**Reply by ettaboyle** [reply]
## Community Team Note: This shipped today 🚀 
I am going to unmark this as the answer on this Discussion and mark another response as the answer to close the loop for anyone who may search and find this Discussion.

Kudos and thanks to @derekprior and team for your work on this and thank you @nedbat for following up with the solution!

**Comment by nedbat** [comment, marked as answer]
The Changelog now mentions this feature: https://github.blog/changelog/2023-02-02-add-more-social-links-to-your-user-profile/

The animation there shows a Mastodon icon for a mastodon.social link.  My link is https://hackyderm.io/@nedbat, which just shows a generic link icon.

The profile docs show a drop-down on the icon (https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#adding-links-to-your-social-accounts), but that's not what I'm seeing.

Is there a way to get a Mastondon icon on the link?

Thanks.

**Reply by ettaboyle** [reply]
## Community Team Note: This shipped today 🚀
I unmarked [another response](https://github.com/orgs/community/discussions/37683#discussioncomment-4430754) as the answer on this Discussion and have marked this response as the answer to close the loop for anyone who may search and find this Discussion.

Kudos and thanks to @derekprior and team for your work on this and thank you @nedbat for following up with the solution!

**Reply by jeroen** [reply]
Would it also be possible to expose the social links in the /user/:username API?

**Comment by maanu1234** [comment]
👍
